packages/util-cli-reporter/lib/js/reporter.js

```diff
@@ -50,27 +54,29 @@ function configureTitle(string) {
  */
 function configureOutput(type, description, message, comment) {
 	return `
-		${colors[type](type)}
-		${colors.description(description)}
-		${(message) ? colors.message(message) : ''}
+		${colors[type](type) + ' '}
```

Comment:
What' the concat with an empty string for? Isn't it the end of the line anyway?

Reply:
I have done this because `message` is optional, so I only want a space there if `message` exists. Because of the way chalk formats stuff, I used to convert linebreaks to spaces, but if comment doesn't exist you end up with two spaces. So now the space is added inline if the variable exists, and all line breaks are just concatenated.

Reply:
it's a bit ugly though, I grant you that. Will see if there is a nicer way to do it